### PR TITLE
Add track-specific docs

### PR DESCRIPTION
**Pull request by angelikatyborska**
According to https://github.com/exercism/docs/blob/aa78d01ec239bab1c9a66a9288e7464ba62ecfbc/anatomy/tracks/shared-files.md#track-specific-files

### CODE REVIEW

File: exercises/shared/.docs/debug.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Debugging
+
+To help with debugging, you can use the fact that all console output will be captured and shown in the test results window. You can output any value by passing it to `IO.inspect`:
+
+```elixir
+"hello"
+|> String.upcase()
+|> IO.inspect()
```
Comment (angelikatyborska) [inline exercises/shared/.docs/debug.md:8]: I think this is the best example of how `IO.inspect` gets used in the wild, but it's not the simplest. Would it be better to give a simpler example, without pipes or the `String` module?

Comment (neenjaw) [reply exercises/shared/.docs/debug.md:8]: or maybe put the inspect between two functions and comment how it allows us to introspect the intermediate value?

**Review by neenjaw — APPROVED** [review]

File: exercises/shared/.docs/tests.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+```elixir
+# @tag :pending
+test "shouting" do
+  assert Bob.hey("WATCH OUT!") == "Whoa, chill out!"
+end
+```
+
+Or, you can enable all the tests by commenting out the
+`ExUnit.configure` line in the test suite.
```
Comment (neenjaw) [inline exercises/shared/.docs/tests.md:26]: Maybe should point them to the `test_helper.exs` file?

Comment (angelikatyborska) [reply exercises/shared/.docs/tests.md]: Good idea, the current wording in misleading.

**Review by neenjaw — APPROVED** [review]

**Pull request merged**